src/utils/vaultMath.ts

```diff
@@ -159,6 +165,7 @@ export const maxCollateralForNewVault = (
     istAvailableBeforeLoanFee.value,
     desiredCollateralization,
     loanFee,
+    true,
```

Comment:
by using the string literal as the option it's readable here what it's doing.
```suggestion
    'ceil',
```

Reply:
Done